website/addons/s3/static/settings.json

```diff
@@ -0,0 +1,13 @@
+{
+    "bucketLocations": {
+        "": "US Standard",
+        "EU": "Europe Standard",
+        "us-west-1": "California",
+        "us-west-2": "Oregon",
+        "ap-northeast-1": "Tokyo",
+        "ap-southeast-1": "Singapore",
+        "ap-southeast-2": "Sydney",
+        "cn-north-1": "Beijing"
```

Comment:
I don't see anything in that issue about Beijing. I know that Frankfurt
requires a v4 signature but it isn't in the list of possible locations for
s3 (yet or possibly ever with boto/boto
<https://github.com/boto/boto/pull/3254>).

On Fri, Jul 31, 2015 at 10:57 AM, Chris Seto <notifications@github.com>
wrote:

> In website/addons/s3/static/settings.json
> <https://github.com/CenterForOpenScience/osf.io/pull/3812#discussion_r35981152>
> :
>
> > @@ -0,0 +1,13 @@
> > +{
> > +    "bucketLocations": {
> > +        "": "US Standard",
> > +        "EU": "Europe Standard",
> > +        "us-west-1": "California",
> > +        "us-west-2": "Oregon",
> > +        "ap-northeast-1": "Tokyo",
> > +        "ap-southeast-1": "Singapore",
> > +        "ap-southeast-2": "Sydney",
> > +        "cn-north-1": "Beijing"
>
> Beijing is currently not supported with boto see:
> jschneier/django-storages#28
> <https://github.com/jschneier/django-storages/issues/28>
> Couldn't this
> <http://docs.pythonboto.org/en/latest/ref/s3.html#boto.s3.regions> be
> used to generate a "bucketlist"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CenterForOpenScience/osf.io/pull/3812/files#r35981152>
> .
>



-- 
*Harry Rybacki* - Developer, Center for Open Science
Research: http://scinet.osf.io
Blog: http://rybacki.io
[image: http://github.com/harryrybacki]
<http://github.com/HarryRybacki> [image:
https://twitter.com/HarryRybacki] <https://twitter.com/HarryRybacki> [image:
http://www.linkedin.com/in/HarryRybacki/]
<http://www.linkedin.com/in/HarryRybacki/> [image:
https://plus.google.com/+HarryRybacki]
<https://plus.google.com/+HarryRybacki>